src/field/field255.rs

```diff
@@ -0,0 +1,483 @@
+
+    fn try_from(value: usize) -> Result<Self, FieldError> {
+        let usize_bytes = value.to_be_bytes();
+        let mut field_bytes = [0u8; 32];
+        let field_bytes_len = field_bytes.len();
+        field_bytes[field_bytes_len - usize_bytes.len()..].copy_from_slice(&usize_bytes);
+        Field255::try_from_bytes(&field_bytes, false)
+    }
+}
+
+impl<'a> TryFrom<&'a [u8]> for Field255 {
+    type Error = FieldError;
+
+    fn try_from(bytes: &[u8]) -> Result<Self, FieldError> {
+        Self::try_from_bytes(bytes, false)
+    }
+}
+
+impl From<Field255> for [u8; Field255::ENCODED_SIZE] {
+    fn from(element: Field255) -> Self {
+        let mut array = [0; 32];
+        fiat_25519_to_bytes(&mut array, &element.0);
+        // The spec calls for big-endian encoding: See
+        // https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/90.
+        array.reverse();
+        array
+    }
+}
+
+impl From<Field255> for Vec<u8> {
+    fn from(elem: Field255) -> Vec<u8> {
+        <[u8; Field255::ENCODED_SIZE]>::from(elem).to_vec()
+    }
+}
+
+impl Display for Field255 {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        let encoded: [u8; Self::ENCODED_SIZE] = (*self).into();
+        write!(f,
+            "0x{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}",
+            encoded[0],
+            encoded[1],
+            encoded[2],
+            encoded[3],
+            encoded[4],
+            encoded[5],
+            encoded[6],
+            encoded[7],
+            encoded[8],
+            encoded[9],
+            encoded[10],
+            encoded[11],
+            encoded[12],
+            encoded[13],
+            encoded[14],
+            encoded[15],
+            encoded[16],
+            encoded[17],
+            encoded[18],
+            encoded[19],
+            encoded[20],
+            encoded[21],
+            encoded[22],
+            encoded[23],
+            encoded[24],
+            encoded[25],
+            encoded[26],
+            encoded[27],
+            encoded[28],
+            encoded[29],
+            encoded[30],
+            encoded[31],
+        )
+    }
+}
+
+impl Debug for Field255 {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        <Self as Display>::fmt(self, f)
+    }
+}
+
+impl Serialize for Field255 {
+    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        let bytes: [u8; Self::ENCODED_SIZE] = (*self).into();
+        serializer.serialize_bytes(&bytes)
+    }
+}
+
+impl<'de> Deserialize<'de> for Field255 {
+    fn deserialize<D: Deserializer<'de>>(deserializer: D) -> Result<Field255, D::Error> {
+        deserializer.deserialize_bytes(FieldElementVisitor {
+            phantom: PhantomData,
+        })
+    }
+}
+
+impl Encode for Field255 {
+    fn encode(&self, bytes: &mut Vec<u8>) {
```

Comment:
This looks nearly identical to the `From<Field255> for [u8; Field255::ENCODED_SIZE]` impl above.